models/sipnet/DESCRIPTION

```diff
@@ -33,3 +33,5 @@ LazyData: FALSE
 Encoding: UTF-8
 Require: 
 RoxygenNote: 7.1.2
+X-schema.org-keywords: R, Predictive Ecosystem Analyzer, Carbon, Water, vegetated ecosystem , atmosphere, high speed model, Bruteforce Suitable, SIPNET
```

Comment:
What is "Bruteforce Suitable"?

Reply:
I read this online "SIPNET is a fast model (∼ 5.5 sec per MCMC iteration in PEcAn including model
execution, and writing and reading model outputs), which makes it suitable for application of bruteforce methods."